src/test/ui/consts/const-eval/ub-incorrect-vtable.rs

```diff
@@ -3,6 +3,15 @@
 //   triggers an error
 // - a similar test that triggers a previously-untested const UB error: emitted close to the above
 //   error, it checks the correctness of the size
+//
+// As is, this code will only hard error when the constants are used, and the errors are emitted via
+// the `#[allow]`-able `const_err` lint. However, if the transparent wrapper technique to prevent
+// reborrows is used -- from `ub-wide-ptr.rs` -- these two errors reach validation and would trigger
+// ICEs as tracked by #86193. So we also use the transparent wrapper to verify actual const UB hard
+// errors are emitted now instead of ICEs.
```

Comment:
```suggestion
// ICEs as tracked by #86193. So we also use the transparent wrapper to verify proper validation
// errors are emitted instead of ICEs.
```